app/views/clusters/_node_status.html.erb

```diff
@@ -12,19 +12,19 @@
 </div>
 <hr>
 <div><h4><%= showqer.available_jobs %> Running or Queued Jobs <br><small>(and <%= showqer.blocked_jobs %> blocked jobs)</small></h4></div>
-<div class="col-sm-2">
-  <div class="pull-right">Running</div>
+
+<div class="row">
+    <div class="col-sm-2 col-xs-3">Running</div>
+    <div class="progress progress-custom col-sm-10 col-xs-9">
+        <div class="progress-bar" role="progressbar" style="width:<%= number_to_percentage(showqer.active_percent*0.88) %>"></div>
+        <div class="col-sm-1 col-xs-1 pull-left" style="padding:0px"><%= showqer.active_jobs %></div>
```

Comment:
Instead of `padding:0px` what if it were something like `padding:0px 5px`. Maybe enforcing a little space between the label and the progress bar in all cases makes sense.

Reply:
Only other comment is that normally the grid classes "col" are directly under a "row"

> Content should be placed within columns, and only columns may be immediate children of rows.

If we remove `col-sm-1 col-xs-1` from the second div, what happens? Is there a sensible style to set to this div that achieves the same goal without the.